### DISCUSSION TITLE
Bundle a theme and commands in one extension or two?

### DISCUSSION THREAD

**Discussion by tonka3000** [microsoft/vscode-discussions#145] [Extension Development QnA] [answered]
Hi,

I have an extension and I wanna provide a color theme as well (the theme is related to the commands, so they are handle the same "product").

**Question: Is it better to bundle the color-theme and the commands in the same extension or should I put the theme into its own extension?**

**Comment by gjsjohnmurray** [comment, marked as answer]
Sounds like the theme has no value other than with the extension whose commands use it, so I'd recommend putting it in that extension.
